enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/catchup/storecopy/StoreCopyClient.java

```diff
@@ -69,70 +75,72 @@ long copyStoreFiles( CatchupAddressProvider catchupAddressProvider, StoreId expe
     private void copyFilesIndividually( PrepareStoreCopyResponse prepareStoreCopyResponse, StoreId expectedStoreId, CatchupAddressProvider addressProvider,
             StoreFileStreams storeFileStreams, Supplier<TerminationCondition> terminationConditions ) throws StoreCopyFailedException
     {
+        ExponentialBackoffStrategy exponentialBackoffStrategy = new ExponentialBackoffStrategy( 1, 1000, TimeUnit.MILLISECONDS );
```

Comment:
I think that the max retires per request is already exposed as a config. This backup strategy is to avoid spamming so I can't see a reason for having it particularly high by default.